runtime/lua/vim/lsp.lua

```diff
@@ -267,14 +267,20 @@ end
 
 --@private
 --- Memoizes a function. On first run, the function return value is saved and
---- immediately returned on subsequent runs.
+--- immediately returned on subsequent runs. If the function returns a multival,
+--- only the first one will be memoized and returned. The function will only be run once
```

Comment:
maybe returned value instead of one

Reply:
And should be a comma after once

Reply:
> maybe returned value instead of one

?

Reply:
If the function returns a multival,only the first {one -> returned value}